feature-libs/product-configurator/rulebased/components/configurator-conflict-and-error-messages/configurator-conflict-and-error-messages.component.ts

```diff
@@ -0,0 +1,40 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { Observable } from 'rxjs';
+import { switchMap } from 'rxjs/operators';
+import { ConfiguratorCommonsService } from '../../core/facade/configurator-commons.service';
+import { Configurator } from '../../core/model/configurator.model';
+import { IconModule } from '@spartacus/storefront';
+import { ConfiguratorRouterExtractorService } from '@spartacus/product-configurator/common';
+
+@Component({
+  selector: 'cx-configuration-conflict-and-error-messages',
+  templateUrl: './configurator-conflict-and-error-messages.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ConfiguratorConflictAndErrorMessagesComponent {
+  iconTypes = IconModule;
```

Comment:
why have this `iconTypes = IconModule;` here?

Reply:
Picked!

Reply:
Done!